components/Navigator.jsx

```diff
@@ -1,58 +1,83 @@
 import React from 'react';
+import { AntDesign } from '@expo/vector-icons';
 import MainView from '../views/MainView';
 import SingleArticleView from '../views/SingleArticleView';
 import SingleCategoryView from '../views/SingleCategoryView';
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 import { useSelector } from 'react-redux';
+import LoginView from '../views/LoginView';
+import store from '../state/store/configureStore';
 
 const Stack = createStackNavigator();
 
 const Navigator = () => {
-  const category = useSelector((state) => state.category);
+  const { category } = useSelector((state) => state);
+
   return (
     <NavigationContainer>
-      <Stack.Navigator>
+      <Stack.Navigator
+        screenOptions={() => ({
+          headerStyle: {
+            backgroundColor: '#111518',
+          },
+          headerTintColor: '#CEC269',
+        })}>
         <Stack.Screen
           name='Home'
           component={MainView}
           options={() => ({
-            ...screenOptions,
-            title: 'Fake ? News',
+            title: 'FAKE ? NEWS',
+            headerTitleStyle: {
+              textAlign: 'center',
+              fontWeight: 'bold',
+            },
+          })}
+        />
+        <Stack.Screen
+          name='logIn'
+          component={LoginView}
+          options={({ navigation }) => ({
+            title: 'Back',
+            headerLeft: () => (
+              <AntDesign
```

Comment:
didnt it had arrows there by default?

Reply:
We customed the arrow on onpress functionality and it takes us directly to home page.